.travis.yml

```diff
@@ -28,7 +28,8 @@ before_install:
 
     # Workaround for pip3. Without the Pyenv line, build fails because both versions 3.6 and 3.6.3 are available.
     - pyenv global system 3.6
-    - pip3 install meson==0.48
```

Comment:
I intentionally specify the versions here, because it's important that the build regularly works with the minimum supported versions.
This goes for `meson` and I see `gcc` below.